automation/taskcluster/androidTest/flank-x86.yml

```diff
@@ -0,0 +1,66 @@
+# gcloud args match the official gcloud cli
+# https://cloud.google.com/sdk/gcloud/reference/firebase/test/android/run
+gcloud:
+  results-bucket: staging.moz-fx-mobile-firebase-testlab.appspot.com
```

Comment:
This is the old storage space.  Needs to be changed to:
results-bucket: focus_android_test_artifacts